tests/component/_task_modules/test_triggers.py

```diff
@@ -1,24 +1,38 @@
-from datetime import timedelta
+from datetime import timezone
 
 import pytest
-from hightime import datetime as ht_datetime
+from hightime import datetime as ht_datetime, timedelta as ht_timedelta
 
-from nidaqmx.constants import Timescale
+import nidaqmx
+from nidaqmx.constants import Edge, Timescale, TimestampEvent, TriggerType
+from nidaqmx.error_codes import DAQmxErrors
 from nidaqmx.task import Task
 
 
 @pytest.fixture()
 def ai_voltage_field_daq_task(task, sim_field_daq_device):
-    """Gets AI voltage task."""
+    """Gets AI voltage field daq task."""
```

Comment:
Update this for https://github.com/ni/nidaqmx-python/pull/513

Reply:
I believe the function name should also be changed to ai_voltage_compact_daq_task since that PR was to replace FieldDAQ with cDAQ. I will create another PR for to address this comment.

Reply:
Hah, woops, sorry for leaving that name confusing! I agree with Brad, no need to be precise here.

Reply:
Addressed in this PR #520